### DISCUSSION TITLE
Running problem deploying Clapshot

### DISCUSSION THREAD

**Discussion by minhsaochen** [elonen/clapshot#74] [General]
I was able to get the container up and running but after logging in with admin/admin.. I get Connecting Server ... spinning wheel and  under view connection error it keeps repeating the following:

Network error fetching 'http://127.0.0.1:8080/api/health

any insight will be greatly appreciated.

**Comment by MindaugasAnusauskas** [comment]
Same issues. Neither chatGPT or Gemini is able to help after 4 hours of testing.

**Comment by elonen** [comment]
Hi! ChatGPT and Gemini probably don't know about enough about the project to help. Bases on previous discussion with similar issues, I used Copilot to cobble up some troubleshooting guidance. You can read them yourself (especially the Architecture Overview) and/or point an LLM to them to help troubleshoot some specific issues.

https://github.com/elonen/clapshot/blob/copilot/fix-fffd24d1-3876-4497-8b5e-e9cfc2b82a20/doc/quick-start-reference.md
https://github.com/elonen/clapshot/blob/copilot/fix-fffd24d1-3876-4497-8b5e-e9cfc2b82a20/doc/connection-troubleshooting.md
https://github.com/elonen/clapshot/blob/copilot/fix-fffd24d1-3876-4497-8b5e-e9cfc2b82a20/doc/architecture-overview.md

Please let me know if these helped or if something is missing, so I know to improve them before merging to main.

**Comment by elonen** [comment]
Here's also a custom GPT that has read all the documentation (including the ones linked above):
https://chatgpt.com/g/g-687debd7cfec8191ad14f604552f0121-clapshot-config-helper
